clients/wallet/src/stellar_wallet.rs

```diff
@@ -305,21 +287,58 @@ impl StellarWallet {
 		Ok(envelope)
 	}
 
+	pub async fn send_payment_to_address_for_redeem_request(
+		&mut self,
+		destination_address: PublicKey,
+		asset: Asset,
+		stroop_amount: StellarStroops,
+		request_id: [u8; 32],
+		stroop_fee_per_operation: u32,
+	) -> Result<TransactionResponse, Error> {
+		let op = self
```

Comment:
Nit: Maybe rename `op` to `payment_like_operation` to indicate that this will be an operation that works similar to a payment? Or we add a comment above that states it will be any of the three types (payment, createaccount, createclaimablebalance)

Reply:
is `payment_like_op` ok?